src/timestamp.h

```diff
@@ -17,8 +17,8 @@
 
 // When considering file modification times we only care to compare
 // them against one another -- we never convert them to an absolute
-// real time.  On POSIX we use time_t (seconds since epoch) and on
-// Windows we use a different value.  Both fit in an int.
-typedef int TimeStamp;
+// real time.  On POSIX we use timespec (seconds&nanoseconds since epoch)
+// and on Windows we use a different value.  Both fit in an int64.
```

Comment:
(...with the Windows change, this would just say something like "File Time":
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724290(v=vs.85).aspx
)